cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py

```diff
@@ -0,0 +1,2650 @@
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_vm_ec2_windows2012_ssui():
+    """
+    1. Provision service with Windows 2012 VM
+    2. Perform SSA on it.
+    3. Check data populated on Provisioned Service in SSUI Dashboard.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        initialEstimate: 1/3h
+        startsin: 5.9
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_vm_ec2_windows2012r2_ntfs():
+    """
+    Add EC-2 provider.
+    Perform SSA on Windows 2012 server R2 VM having NTFS filesystem.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.9
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_vm_azure_rhel():
+    """
+    Create or Use existing RHEL VM/Instance present in Azure.
+    Perform SSA on RHEL VM/Instance when
+    VM/Instance is Powered ON
+    VM/Instance is Powered OFF
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        initialEstimate: 1/4h
+        startsin: 5.6
+        upstream: no
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_vm_azure_wimdows2016_ssui():
+    """
+    1. Provision service with Windows 2016 VM
+    2. Perform SSA on it.
+    3. Check data populated on Provisioned Service in SSUI Dashboard.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        initialEstimate: 1/3h
+        startsin: 5.8
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_vm_azure_windows2016_refs():
+    """
+    This test is to verify that you get an error when trying to perform
+    SSA on a Windows2016 VM that has a ReFS formatted disk attached.  Here
+    is the before and after.
+    05/26/17 18:13:36 UTC
+    05/26/17 18:10:56 UTC
+    05/26/17 18:10:46 UTC
+    finished
+    Unable to mount filesystem. Reason:[ReFS is Not Supported]
+    Scan from Vm ReFS16on16a
+    admin
+    EVM
+    Scanning completed.
+    05/26/17 16:12:45 UTC
+    05/26/17 16:08:30 UTC
+    05/26/17 16:08:26 UTC
+    finished
+    Process completed successfully
+    Scan from Vm ReFS16on16a
+    admin
+    EVM
+    Synchronization complete
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/4h
+        setup: Need to create a windows 2016 instance and save it onto the ReFS disks
+               on Azure.
+               Add a second disk to the VM config.  Open VM, initialize disk and
```

Comment:
```setup:``` should be like
```
 setup:
     1. Need to create a windows 2016 instance and save it onto the ReFS disks
         on Azure.
     2.
```

Reply:
This is present for almost all tests, I will re-format it while automating them.